src/apps/js_v8/v8_oe_stubs.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#include "ds/logger.h"
+
+#include <pthread.h>
+#include <semaphore.h>
+#include <signal.h>
+#include <stdio.h>
+#include <sys/resource.h>
+
+// This file stubs the following functions used by V8 that
+// are not available in the Open Enclave standard libraries.
+// Stubbing them is ok because they are not used by V8, mostly
+// because we disable threading.
+
+/*
```

Comment:
??

Reply:
It's just a little dump so that anyone curious can see where those symbols are used in the code. Maybe it's useless?

Reply:
I normally do that with a short description on the top and a more edited version, but if people are fine with this, no worries.